src/mux-analytics.brs

```diff
@@ -339,6 +357,29 @@ function muxAnalytics() as Object
     m._videoSourceDuration = Invalid
     m._viewPrerollPlayedCount = Invalid
 
+    m._lastSourceWidth = Invalid
+    m._lastSourceHeight = Invalid
+    m._lastPlayheadPosition = Invalid
+    m._lastVideoSegmentBitrate = Invalid
+    m._viewMaxUpscalePercentage = Invalid
+    m._viewMaxDownscalePercentage = Invalid
+    m._viewTotalUpscaling = Invalid
+    m._viewTotalDownscaling = Invalid
+    m._viewTotalContentPlaybackTime = Invalid
+    m._totalBytes = Invalid
+    m._totalLoadTime = Invalid
+    m._segmentRequestCount = Invalid
+    m._segmentRequestFailedCount = Invalid
+    m._viewMinRequestThroughput = Invalid
+    m._viewAverageRequestThroughput = Invalid
+    m._viewRequestCount = Invalid
+
+    ' Calculate player width and heitht
+    deviceInfo = m._getDeviceInfo()
+    videoMode = deviceInfo.GetVideoMode()
```

Comment:
I think this is fine, but just to make sure, this info is all available at the initial setup time?

Reply:
Yes, setting values during initial setup.